Bottom: c70c010391f70af249bd633474ce236bcdc34133
Top:    bf1955f26d051e41cdd7d1c4b5dd7ff4208687fb
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 08:25:45 -0500

Remove container conf directory

Roles create it themselves


---
diff --git a/roles/docker-common/tasks/main.yml b/roles/docker-common/tasks/main.yml
index 396b842c..7ef3f40c 100644
--- a/roles/docker-common/tasks/main.yml
+++ b/roles/docker-common/tasks/main.yml
@@ -8,13 +8,6 @@
 - import_tasks: sshd_settings.yml
 - import_tasks: mariadb_settings.yml
 
-- name: base docker_container.conf dir
-  file:
-    state: directory
-    path: '{{ docker_restore_config_base_dir }}'
-    owner: root
-    group: root
-    mode: 'u=rwx,g=rx,o=rx'
 
 
 - name: Check if baculavirt selinux module exists
